form/form_themes.rst

```diff
@@ -371,6 +371,65 @@ by ``row``, ``widget``, ``label``, or ``help``):
         {# the field of TaskType #}
     {% endblock %}
 
+Access data in a Form Choice
+.............................
+
+When you have a ChoiceType or an EntityType, and you want to customize the ``entry``, row or widget, you can also retrieve
+the data of the choice, but not directly in the ``form.vars.data`` of the child. See an exemple::
+
+    class TaskManagerType extends AbstractType
```

Comment:
By moving this section as suggested above, adding one more PHP example should not be necessary in that context, a reference to https://symfony.com/doc/4.4/reference/forms/types/choice.html#advanced-example-with-objects could be enough as it is also used to illustrate most of the options in that page.